CMakeLists.txt

```
@@ -28,6 +28,7 @@ option(ENABLE_LIBCXX "Enables LLVM libc++" FALSE)
 option(ENABLE_CCACHE "Enables ccache for compile caching" TRUE)
 option(ENABLE_VIXL_SIMULATOR "Forces the FEX JIT to use the VIXL simulator" FALSE)
 option(ENABLE_VIXL_DISASSEMBLER "Enables debug disassembler output with VIXL" FALSE)
+option(ENABLE_SYSTEMD_BINFMTMISC "Enables the systemd binfmt_misc install targets" TRUE)
```

For better or worse, #3908 kicked off a route where options that trigger mutually exclusive paths should use a `USE_` prefix instead of `ENABLE_`.

Subjectively these options should be named such that they are FALSE by default as well (to better indicate what a "normal" setup looks like), so in this case it would be something like:

```suggestion
option(USE_LEGACY_BINFMTMISC "Uses legacy method of setting up binfmt_misc" FALSE)
```

---

Sure, renamed.